ansible/roles/flannel/handlers/main.yml

```diff
@@ -14,8 +35,21 @@
   command: ip link delete docker0
   when: '"docker0" in ansible_interfaces'
 
-- name: start docker
+  # TODO: work only on default installation...
+- name: docker graph driver 
+  command: findmnt -no FSTYPE -T /var/lib/docker
```

Comment:
`docker0` rise only after docker successfully start-up, so we **can** use it in stop/remove tasks and **can't** in fix/start (first run)